package.json

```diff
@@ -5,15 +5,23 @@
   "author": "Government Digital Service",
   "license": "MIT",
   "scripts": {
-    "lint": "standard app/assets/javascripts/**/*",
-    "lint:fix": "standard --fix app/assets/javascripts/**/*"
+    "lint": "standard app/javascript/{*,**/*}",
```

Comment:
When I do `ls **/*` I get the top-level files. Was there a problem with this here?

Reply:
Yeah there was unfortunately - I think it ties into what exactly does the globbing.

e.g. if I do:

```
vm  content-publisher git:(yarn) echo "withSemiColon();\n" >> app/javascript/restrict.js
```

then with os globbing:

```
vm  content-publisher git:(yarn) ✗ yarn run standard app/javascript/**/*
yarn run v1.9.4
$ /var/govuk/content-publisher/node_modules/.bin/standard app/javascript/components app/javascript/components/contextual-guidance.js app/javascript/components/markdown-editor.js app/javascript/documents app/javascript/documents/edit.js app/javascript/packs app/javascript/packs/application.js app/javascript/restrict.js
standard: Use JavaScript Standard Style (https://standardjs.com)
standard: Run `standard --fix` to automatically fix some problems.
  /var/govuk/content-publisher/app/javascript/restrict.js:6:1: 'withSemiColon' is not defined.
  /var/govuk/content-publisher/app/javascript/restrict.js:6:16: Extra semicolon.
  /var/govuk/content-publisher/app/javascript/restrict.js:7:1: Too many blank lines at the end of file. Max of 0 allowed.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

but relying on the config file:

```
vm  content-publisher git:(yarn) ✗ yarn run lint
yarn run v1.9.4
$ standard app/javascript/**/*
Done in 1.39s.
```

however with the `{*,*/*}` change:

```
vm  content-publisher git:(yarn) ✗ yarn run lint
yarn run v1.9.4
$ standard app/javascript/{*,**/*}
standard: Use JavaScript Standard Style (https://standardjs.com)
standard: Run `standard --fix` to automatically fix some problems.
  /var/govuk/content-publisher/app/javascript/restrict.js:6:1: 'withSemiColon' is not defined.
  /var/govuk/content-publisher/app/javascript/restrict.js:6:16: Extra semicolon.
  /var/govuk/content-publisher/app/javascript/restrict.js:7:1: Too many blank lines at the end of file. Max of 0 allowed.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

I'll update the commit though to make it clearer that this was actually a wtf